atlasdb-dagger/src/main/java/com/palantir/atlasdb/services/RawKeyValueServiceModule.java

```diff
@@ -17,18 +17,56 @@
 
 import com.palantir.atlasdb.keyvalue.api.KeyValueService;
 import com.palantir.atlasdb.util.MetricsManager;
+import com.palantir.logsafe.logger.SafeLogger;
+import com.palantir.logsafe.logger.SafeLoggerFactory;
 import dagger.Module;
 import dagger.Provides;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
 import javax.inject.Named;
 import javax.inject.Singleton;
 
 @Module
 public class RawKeyValueServiceModule {
+    private static final SafeLogger log = SafeLoggerFactory.get(RawKeyValueServiceModule.class);
+    private static final ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
```

Comment:
nit: prefer `PtExecutors` (it does some clever magic to reuse threads between executors)